lib/clis/wolkenkit/createDeployment/createDeploymentCommand.ts

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env node
+
+import { buntstift } from 'buntstift';
+import { Command } from 'command-line-interface';
+import { createDeploymentManifests } from './createDeploymentManifests';
+import { CreateDeploymentOptions } from './CreateDeploymentOptions';
+import { exists } from '../../../common/utils/fs/exists';
+import { getAbsolutePath } from '../../../common/utils/path/getAbsolutePath';
+import { nameRegularExpression } from '../init/nameRegularExpression';
+import path from 'path';
+import shell from 'shelljs';
+import { validateName } from '../init/validateName';
+
+const createDeploymentCommand = function (): Command<CreateDeploymentOptions> {
+  return {
+    name: 'init',
+    description: 'Initialize a new application.',
```

Comment:
I assume this is a left-over from copy/paste and needs to be adjusted.

```suggestion
    name: 'create-deployment',
    description: 'Create deployment manifests.',
```

Reply:
Yes, I missed those.